src/go/k8s/Makefile

```diff
@@ -73,24 +78,31 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build -f Dockerfile -t ${IMG} ../
-
-# Push the docker image
-docker-push:
-	docker push ${IMG}
+	docker build -f Dockerfile -t ${OPERATOR_IMG_LATEST} ../
```

Comment:
thought for next PRs: we could build both images on a single target when testing/releasing.